common/dal/src/accessors/aws_s3/s3_input_stream.rs

```diff
@@ -102,9 +101,14 @@ impl futures::AsyncRead for S3InputStream {
                             counter!(super::metrics::METRIC_S3_GETOBJECT_ERRORS, 1);
                             match e {
                                 RusotoError::Service(GetObjectError::NoSuchKey(msg)) => {
+                                    dbg!("not found");
                                     Error::new(ErrorKind::NotFound, msg)
                                 }
-                                _ => Error::new(ErrorKind::Other, e),
+                                _ => {
+                                    dbg!("other");
```

Comment:
Why not use `tracing::error!()`?

Reply:
oh, that's WIP version, it is removed in commit 4ba45b8 (tidy up)

Reply:
Hmm, assuming this is a debug print, please ignore the previous comments